requirements.txt

```diff
@@ -1,5 +1,6 @@
 # For real.
-pueblo==0.0.3
+# pueblo==0.0.3
+poethepoet
 
 # Development.
-# pueblo @ git+https://github.com/pyveci/pueblo.git@develop
+pueblo @ git+https://github.com/pyveci/pueblo.git@ngr-python-poe
```

Comment:
This must converge back to a GA release, i.e. c703d403305 and 7cf85e08a78e must be partially reverted.